secantTests/SendTests/SendTests.swift

```diff
@@ -724,13 +727,16 @@ class SendTests: XCTestCase {
         let store = TestStore(
             initialState: sendState
         ) {
-            SendFlowReducer(networkType: .testnet)
+            SendFlowReducer()
         }
         
         XCTAssertEqual(store.state.message, testMessage.data)
     }
     
     func testFeeFormat() throws {
+        let zashiBalanceFormatter = NumberFormatter.zashiBalanceFormatter
+        zashiBalanceFormatter.locale = Locale(identifier: "en_US")
```

Comment:
Ditto.

Reply:
Yes, the simulators have different locale settings and tests were failing. Also CIs use different version of xcode and simulators so I needed to setup the tests with hardcoded locale to US (sometime to CZ so I can check comma instead of point as a floating separator).